gradle/core.versions.toml

```diff
@@ -24,22 +24,22 @@ kotlinxCoroutines = "1.10.2"
 kotlinxSerialization = "1.8.1" # TODO: Update to 1.9.0 or later after we bump Kotlin to 2.2.0
 ktlintPlugin = "13.1.0"
 ktor = "3.2.3"
-lifecycleRuntimeKtx = "2.9.2"
-lottie = "6.6.7"
+lifecycleRuntimeKtx = "2.9.4"
+lottie = "6.6.10"
 material = "1.12.0"
 matomo = "4.3.4"
-navigation = "2.9.3"
+navigation = "2.9.5"
 okhttp = "4.12.0" # TODO: Update to 5.0.0 or later after we bump Kotlin to 2.2.0
 playReview = "2.0.2"
 progressButton = "2.1.0"
 recyclerView = "1.3.2" # TODO: Update to 1.4.0 or later after we bump compileSdk to 35
 robolectric = "4.13" # Don't update this because it breaks tests
 room = "2.7.2"
-sentry = "8.21.1"
+sentry = "8.23.0"
```

Comment:
In kDrive :
_java.lang.IllegalStateException: Sentry SDK has detected a mix of versions. This is not supported and likely leads to crashes. Please always use the same version of all SDK modules (dependencies). See https://docs.sentry.io/platforms/android/troubleshooting/mixed-versions for more details._

Reply:
_The Sentry SDK has been configured with mixed versions. Expected maven:io.sentry:sentry-android-timber to match core SDK version 8.23.0 but was 8.21.1_

Reply:
If you bump the Sentry plugin in kDrive from `5.11.0` to `5.12.1` and you downgrade Sentry from `8.23.0` to `8.22.0`, it fixes this issue.

Reply:
The best solution is probably to remove the Sentry plugin from kDrive.
We don't have in in kMail. The Sentry configuration is done differently there.